Project-I-Social-Card/social-card/src/components/CardComponents/Card.css

```
@@ -1 +1,46 @@
 /* All styles for your Card Components live here */
+
+/* .cardContainer {
+    text-decoration: none;
+    color: black;
+    border: 1px solid black;
+    padding: 10 0px 0px 0px;
+} */
+
+.container {
+    border: 1px solid lightgrey;
+    float: right;
+    max-width: 90%;
+    border-radius: 3px;
+    margin-right: 4%;
+}
+
+@media(max-width: 600) {
+    .container {
+        border: 1px solid lightgrey;
+        float: right;
+        max-width: 90%;
+        border-radius: 3px;
+        /* margin-right: -1%; */
+    }
+}
+
+/* React logo image*/
+
+.cardBanner {
```

Industry standard naming convention for css declarations is lower-case words separated by a hyphen. So this declaration would look like this:

```suggestion
.card-banner {
```

Try to industry standard convention in future projects.